config/bto.yml

```diff
@@ -4,8 +4,25 @@ idspace: BTO
 base_url: /obo/bto
 
 products:
-- bto.owl: http://ontologies.berkeleybop.org/bto.owl
+- bto.owl: https://github.com/BRENDA-Enzymes/BTO/raw/master/bto.owl
+- bto.obo: https://github.com/BRENDA-Enzymes/BTO/raw/master/bto.obo
+- bto.json: https://github.com/BRENDA-Enzymes/BTO/raw/master/bto.json
 
 term_browser: ontobee
 example_terms:
 - BTO_0000000
+
+entries:
+
+- prefix: /releases/
+  replacement: https://github.com/BRENDA-Enzymes/BTO/v
```

Comment:
I was expecting this to point to a raw file, not HTML.

Reply:
An that is why i still need people looking over my shoulder. Thanks for the diligence @jamesaoverton should be fixed now!

Reply:
No problem. I approve this. Please merge when you're ready.